test/metabase/test_setup.clj

```diff
@@ -99,3 +99,17 @@
   []
   (log/info "Shutting down Metabase unit test runner")
   (core/stop-jetty!))
+
+(defn call-with-test-scaffolding
```

Comment:
Is this used anywhere? Or if it's only for REPL usage can we also make it `^:private`?